common/djangoapps/student/models.py

```diff
@@ -233,7 +233,10 @@ class UserProfile(models.Model):
 
     class Meta(object):
         db_table = "auth_userprofile"
-        permissions = (("can_deactivate_users", "Can deactivate, but NOT delete users"),)
+        permissions = (
+            ("can_deactivate_users", "Can deactivate, but NOT delete users"),
+            ("can_call_check_course_access_api", "Can access CheckCourseAccessView"),
```

Comment:
This permission will allow a user to access the embargo api and the permission will be assigned to a user. Is there a reason why it should be moved?